tests/baselines/reference/user/lodash.log

```diff
@@ -29,6 +30,7 @@ node_modules/lodash/_asciiWords.js(8,20): error TS8024: JSDoc '@param' tag has n
 node_modules/lodash/_baseClone.js(91,16): error TS2362: The left-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.
 node_modules/lodash/_baseClone.js(92,16): error TS2362: The left-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.
 node_modules/lodash/_baseClone.js(93,16): error TS2362: The left-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.
+node_modules/lodash/_baseClone.js(114,18): error TS2554: Expected 0 arguments, but got 1.
```

Comment:
The type of function expressions in conditional expressions has changed. Might be ones that are initialisers of variable decls with jsdoc, not sure yet.

Reply:
Here's a repro. I guess we changed our rules for unioning signatures recently.

```js
var noop = () => { };

/**
 * @returns {void}
 */
var createSet = !!false ? noop : function(values) {
};
createSet(1) // error before, fine now
```